stellargraph/layer/gcn.py

```diff
@@ -56,47 +56,42 @@ class GraphConvolution(Layer):
         use_bias (bool): toggles an optional bias
         final_layer (bool): If False the layer returns output for all nodes,
                             if True it returns the subset specified by the indices passed to it.
-        kernel_initializer (str): name of layer bias f the initializer for kernel parameters (weights)
-        bias_initializer (str): name of the initializer for bias
-        attn_kernel_initializer (str): name of the initializer for attention kernel
-        kernel_regularizer (str): name of regularizer to be applied to layer kernel. Must be a Keras regularizer.
-        bias_regularizer (str): name of regularizer to be applied to layer bias. Must be a Keras regularizer.
-        activity_regularizer (str): not used in the current implementation
-        kernel_constraint (str): constraint applied to layer's kernel
-        bias_constraint (str): constraint applied to layer's bias
+
+    Optional Args:
+        kernel_initializer (str or func): The initialiser to use for the weights.
+        kernel_regularizer (str or func): The regulariser to use for the weights.
+        kernel_constraint (str or func): The constraint to use for the weights.
+        bias_initializer (str or func): The initialiser to use for the bias.
+        bias_regularizer (str or func): The regulariser to use for the bias.
+        bias_constraint (str or func): The constraint to use for the bias.
     """
 
     def __init__(
-        self,
-        units,
-        activation=None,
-        use_bias=True,
-        final_layer=False,
-        kernel_initializer="glorot_uniform",
-        bias_initializer="zeros",
-        kernel_regularizer=None,
-        bias_regularizer=None,
-        activity_regularizer=None,
-        kernel_constraint=None,
-        bias_constraint=None,
-        **kwargs
+        self, units, activation=None, use_bias=True, final_layer=False, **kwargs
```

Comment:
I'm not sure I like the use of `**kwargs` here, it makes it harder for users to know what the arguments can be. Why not list them all?

Reply:
Partly because the auto code checker doesn't like too many arguments (for good reason), and partly for the same reason you mention above - we can have additional (non-typical) arguments for expert users that are not mentioned in the method docs, to avoid confusing the typical user.

Reply:
I'm happy with that 👍